### PR TITLE
doc/http2: suricata.yaml max-streams parameter

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/4949

Describe changes:
- Adds documentation about suricata.yaml `http2.max-streams` parameter

### CODE REVIEW

File: doc/userguide/configuration/suricata-yaml.rst [hunk 1]
```diff
@@ -1725,6 +1725,11 @@ An app-layer event `protocol.too_many_transactions` is triggered when this value
 The point of this parameter is to find a balance between the completeness of analysis
 and the resource consumption.
 
+For HTTP2, this parameter is named `max-streams` as a HTTP2 stream will get translated
+into one Suricata transaction. This configuration parameter is used whatever the
+value of `SETTINGS_MAX_CONCURRENT_STREAMS` negotiated between a client and a server
+in a specific flow.
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:1731]: Just two suggestions :)
```diff
- For HTTP2, this parameter is named `max-streams` as a HTTP2 stream will get translated
- into one Suricata transaction. This configuration parameter is used whatever the
- value of `SETTINGS_MAX_CONCURRENT_STREAMS` negotiated between a client and a server
- in a specific flow.
+ For HTTP2, this parameter is named `max-streams` as an HTTP2 stream will get translated
+ into one Suricata transaction. This configuration parameter is used whatever the
+ value of `SETTINGS_MAX_CONCURRENT_STREAMS` negotiated between a client and a server
+ in a specific flow is.
```
(an HTTP2)
(flow is)

**Review by jufajardini — CHANGES_REQUESTED** [review]
I left a suggestion for small changes to improve readability :)

File: doc/userguide/configuration/suricata-yaml.rst (hunk 1 above)
Comment (catenacyber) [reply doc/userguide/configuration/suricata-yaml.rst:1731]: Thanks

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata/pull/7813

**Pull request closed without merging**